python/src/EmpiricalBernsteinCopula_doc.i.in

```diff
@@ -4,11 +4,17 @@
 Parameters
 ----------
 sample : :class:`~openturns.Sample`
-    The sample of size :math:`N>0` and dimension :math:`d` from which the empirical copula sample is extracted. Default is *sample=[[0.0], [0.0]]*.
+    The sample of size :math:`N>0` and dimension :math:`d` from which the empirical copula sample is extracted.
+    
+    Default value is :math:`[[0.0], [0.0]]`.
```

Comment:
The current rendering is:
![image](https://github.com/openturns/openturns/assets/31351465/300a64eb-f981-4d71-b6cb-2b4fc50cbdfd)

The suggested rendering is:
![image](https://github.com/openturns/openturns/assets/31351465/92ac5223-6182-4495-924c-c2c594f0ba50)

Actually, I rather have the current rendering, because it refers to a Python code.

Reply:
ok. To get the suggested rendering, I wrote: 
Default value is :math:`[[0.0], [0.0]]`.